src/main/java/com/google/codeu/servlets/UserLevelServlet.java

```diff
@@ -95,4 +97,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             response.sendRedirect("/puzzle.html?user=" + user_email);
         }
     }
+
+    /*strips the user answer os all punctuation, spaces, and any "an" "a" or "the" 
+    assumes you pass in answer in all caps
+    */
+    public String stripAnswer(String userAnswer){
+        String result = "";
+        String firstWord = userAnswer.substring(0, userAnswer.indexOf(' '));
+
+        //if the first word of the user answer equals an, a, or the, remove from string
+        if(firstWord.equals("AN") || firstWord.equals("A") || firstWord.equals("THE")){
+            userAnswer = userAnswer.substring(userAnswer.indexOf(' ') + 1);
+        }
+        //go through user answer to sript any punctation 
+        for(int i = 0; i < userAnswer.length(); i++){
+            char letter = userAnswer.charAt(i);
+            //check if character is letter or digit 
+            if(Character.isLetterOrDigit(letter)){
+                result += letter;
+            }
+        }
+        result = result.trim();
+        return result;
+    }
```

Comment:
looks good but see if there is a more concise way of doing it (either with string methods or reg exp)